src/components/UiKit/ScreenBackground.js

```diff
@@ -1,19 +1,13 @@
 // @flow
 
-import React from 'react'
+import React, {type ComponentType} from 'react'
```

Comment:
Minor detail: According to our Style guide we group type instantiations at the end (after styles). No need to change it but to keep in mind. See README for more details